packages/builder/src/components/portal/overview/automation/HistoryTab.svelte

```diff
@@ -145,6 +159,18 @@
           options={statusOptions}
         />
       </div>
+      {#if isEnabled(TENANT_FEATURE_FLAGS.LICENSING) && licensePlan?.type === "free"}
+        <div class="pro-upgrade">
+          <div class="pro-copy">Store up to 30 days of automations</div>
+          <Button
```

Comment:
Similar to the other checks, in cloud we should only enable the upgrade button to account holders. In self host we can leave it on always. For the groups upgrade button I did this: https://github.com/Budibase/budibase/blob/master/packages/builder/src/pages/builder/portal/manage/groups/index.svelte#L125